patterns/pattern-focil-eip7805.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Pattern: FOCIL - Fork-Choice Enforced Inclusion Lists"
+status: draft
+maturity: PoC
+works-best-when:
+  - Censorship resistance is critical
+  - Block builders concentrate power and may censor transactions
+  - Same-slot transaction inclusion guarantees needed
+avoid-when:
+  - Mempool visibility of transaction details is unacceptableh
+dependencies: [EIP-7805]
+---
+
+## Intent
+
+FOCIL (Fork-Choice Enforced Inclusion Lists) is a standard that enables a committee of 16 validators to force-include transactions in blocks, preventing censorship by sophisticated block builders. FOCIL itself does not provide privacy, transactions remain fully visible in the public mempool and on-chain. It only guarantees that transactions cannot be censored once submitted to the mempool.
```

Comment:
Might want to say EIP standard to be explicit (assuming all captured in it)

Reply:
A section on timeline for rollout would be good. E.g. schedueld for X hard fork, or TBD (ditto below)